core/templates/dev/head/editor/EditorServices.js

```diff
@@ -215,6 +241,18 @@ oppia.factory('changeListService', [
     }
     explorationChangeList.push(changeDict);
     undoneChangeStack = [];
+
+    // FIXME: Uncomment this block when the autosave PUT request url is fixed.
```

Comment:
We don't include commented out code in commits as it tends to not get maintained properly and then causes problems later. You can save it in a text document on your computer somewhere and then remove it here.